### PR TITLE
Migrate test packages

### PR DESCRIPTION
**Pull request by jgoshi**
@piotrpMSFT @livarcocc @krwq

### CODE REVIEW

**Comment by jgoshi** [comment]
test Windows_NT x86 Debug Build

**Comment by jgoshi** [comment]
test OpenSUSE13.2 x64 Debug Build

File: build/test/TestPackageProjects.targets [hunk 1]
```diff
@@ -71,54 +72,62 @@
...
         <Frameworks>net451;netcoreapp1.0</Frameworks>
+        <PackRuntime>$(CoreCLRRid)</PackRuntime>
       </BaseTestPackageProject>
       <BaseTestPackageProject Include="TestAssets/TestPackages/dotnet-desktop-binding-redirects"
                           Condition=" '$(IsDesktopAvailable)' == 'True' ">
         <Name>dotnet-desktop-binding-redirects</Name>
+        <ProjectName>dotnet-desktop-binding-redirects.csproj</ProjectName>
         <IsTool>True</IsTool>
-        <IsMsbuild>True</IsMsbuild>
+		<IsMsbuild>True</IsMsbuild>
```
Comment (piotrpMSFT) [inline build/test/TestPackageProjects.targets]: ???

Comment (jgoshi) [inline build/test/TestPackageProjects.targets]: Will fix that.

Comment (jgoshi) [inline build/test/TestPackageProjects.targets]: In a separate one (include with my next change) so I don't have to wait for tests again before merging.

**Comment by jgoshi** [comment]
test Windows_NT x86 Debug Build

**Comment by jgoshi** [comment]
test Windows_NT x86 Debug Build

**Comment by jgoshi** [comment]
@dotnet-bot test Windows_NT x86 Debug Build

**Pull request merged**